README.md

```diff
@@ -1,31 +1,56 @@
 # Blockchain Shop
 
 ## Dependencies
+
 * Docker
+* Kubernetes (using kind - Kubernetes in Docker)
+
+## Getting Started
+
+This repository sets up a Kubernetes cluster with a backend written in Go and a frontend written in Angular. Follow the steps below to get started.
+
+### Create the Cluster
+
+First, navigate to the `deployment` directory and run the `cluster.sh` script to create the cluster.
 
-## How to use repo
-In this repository you are going to create a kind (Kubernetes in Docker) cluster with a backend based in Go and a frontend based in angular
-First of all you should create the cluster
 ```bash
 cd deployment
 ./cluster.sh
 ```
 
-At this moment you have a cluster with 1 master and 2 nodes. All staff is deployed with the right configuration.
-Go to http://localhost:30000/swagger/index.html to see the backend API documentation.
+This script will create a Kubernetes cluster with 1 master node and 2 worker nodes. All components will be deployed with the correct configurations.
+
+### Accessing the Backend API Documentation
+
+Once the cluster is up and running, you can access the backend API documentation at:
+
+[http://localhost:30000/swagger/index.html](http://localhost:30000/swagger/index.html)
```

Comment:
At the end, the real path to the API documentation is:
[http://localhost:30000/api/swagger/index.html](http://localhost:30000/api/swagger/index.html)
Could you change it in your branch?

Reply:
Done! Thanks for letting me know about that detail!

Reply:
Thanks for help us.